app/models/folio/bib_data.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Folio
+  # Bibliographic data from Folio.
+  class BibData
+    def initialize(request, _ = nil)
+      @request = request
+    end
+
+    attr_reader :request
+
+    def title
+      json.fetch('indexTitle', '')
+    end
+
+    def author
```

Comment:
Do we need to filter this list at all? I'm trying to find the symphony equivalent, but it'd surprise me if we show every contributor (but maybe FOLIO means something different?)

Reply:
I don't know the answer to this. Seems like a question for @saseestone

Reply:
I think this is a parity question. Searchworks is pulling the only first author from some very specific fields:

```
document['author_person_display']&.first ||
        document['author_corp_display']&.first ||
        document['author_meeting_display']&.first,
 ```

I see contributors have a `primary` attribute that might be useful for filtering without needing to replicate the MARC 100/110/111 behavior, but we'd need to do some due diligence to find out if it's similar.

Reply:
Filtered by "primary" now.

Reply:
Filtering by `primary` seems to get us the right contributors, at least the few I've spot-checked.